docs/sdks/uid2-sdk-ref-mobile-ima.md

```diff
@@ -0,0 +1,113 @@
+# Developer Guide for UID2 IMA Plugin (iOS and Android)
```

Comment:
Merely converting the word doc to markdown. didn't update the content to latest as Google isn't ready to release GMA fixes yet.